Makefile

```diff
@@ -3,18 +3,18 @@
 #See clock.c for the license detail.
 
 SRC = ttyclock.c
-CC ?= gcc
+CC += gcc
 BIN = tty-clock
-PREFIX ?= /usr/local
+PREFIX += /usr/local
 INSTALLPATH = ${DESTDIR}${PREFIX}/bin
 MANPATH = ${DESTDIR}${PREFIX}/share/man/man1
 
 ifeq ($(shell sh -c 'which ncurses5-config>/dev/null 2>/dev/null && echo y'), y)
-	CFLAGS ?= -Wall -g -I $$(ncurses5-config --includedir)
-	LDFLAGS ?= -L $$(ncurses5-config --libdir) $$(ncursesw5-config --libs)
+	CFLAGS += -Wall -g -I $$(ncurses5-config --includedir)
+	LDFLAGS += -L $$(ncurses5-config --libdir) $$(ncursesw5-config --libs)
 else ifeq ($(shell sh -c 'which ncursesw5-config>/dev/null 2>/dev/null && echo y'), y)
-		CFLAGS ?= -Wall -g -I $$(ncursesw5-config --includedir)
-		LDFLAGS ?= -L $$(ncursesw5-config --libdir) $$(ncursesw5-config --libs)
+		CFLAGS += -Wall -g -I $$(ncursesw5-config --includedir)
+		LDFLAGS += -L $$(ncursesw5-config --libdir) $$(ncursesw5-config --libs)
```

Comment:
in the case of *FLAGS, I could be convinced otherwise - especially for CFLAGS... in most cases, however, `ncurses-config --cflags` (as it current is written) is mostly a no-op, so overriding is fine there. overriding LDFLAGS globally is shooting yourself in the foot, however - not sure how this works with other software...?

but i could be convinced otherwise. in any case, this would need a rebase.